client/landing/gutenboarding/onboarding-block/site-title.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { useDispatch, useSelect } from '@wordpress/data';
+import React from 'react';
+import { __ as NO__ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { STORE_KEY } from '../stores/onboard';
+import { StepInputProps } from './stepper-wizard';
+import Question from './question';
+
+export default function SiteTitle( { onSelect, inputClass, ...props }: StepInputProps ) {
+	const { siteTitle } = useSelect( select => select( STORE_KEY ).getState() );
+	const { setSiteTitle } = useDispatch( STORE_KEY );
+
+	const handleChange = ( e: React.ChangeEvent< HTMLInputElement > ) =>
+		setSiteTitle( e.target.value );
+
+	const label = NO__( "It's called" );
+	const value = siteTitle.length ? siteTitle : NO__( 'enter a title' );
+
+	return (
+		<>
+			<Question label={ label } displayValue={ value } { ...props }>
+				<input
+					className={ inputClass }
+					placeholder={ NO__( 'enter a title' ) }
+					onChange={ handleChange }
+					onBlur={ onSelect }
+					value={ siteTitle }
```

Comment:
The duplication of values that we pass both to `<Question />` _and_ to `<input />` here (the `placeholder` string, and `value`) kind of seems like evidence for [this hypothesis](https://github.com/Automattic/wp-calypso/pull/37491/files#r346310923) to me :thinking:

Reply:
The placeholder of the input and the empty `displayValue` passed to `Question` are not necessary the same (otherwise we'd be just using a constant for both). The former should help the user better understand what to time inside the input (with examples maybe) while the later is more like a call to action.